.travis.yml

```diff
@@ -4,26 +4,44 @@ sudo: false
 
 cache:
     directories:
-        - bin
-        - vendor
+        - ~/.composer/cache/files
 
 php:
-    - 7.2
     - 7.1
+    - 7.2
 
 env:
-    matrix:
-        - PREFER_LOWEST=""
-        - PREFER_LOWEST="--prefer-lowest"
-
-matrix:
-    fast_finish: true
+    - SYMFONY_VERSION=3.4.*
+    - PHPUNIT_VERSION=^6.0
+    - CODUO_VERSION=^2.3
+    - MOST_RECENT=true
```

Comment:
Not needed atm

Reply:
Why? It run tests with Sf 4, PHPUnit 7 and Coduo 3 at the same time.